### DISCUSSION TITLE
use reply as input for text based commands

### DISCUSSION THREAD

**Discussion by TheScreechingBagel** [machineonamission/mediaforge#243] [Ideas]
commands that take text for input (like $1984 , $eminem etc) should check if the command is sent as a reply to another message, and use that message as input if no other input is specified. This would be similar to the current behaviour of image commands, where if you reply with `$caption <text>` to an image, that image is used

**Comment by machineonamission** [comment]
i don't hate the idea but im not sure discord.py would like this, it'd be a headache to implement

**Reply by coolerrat** [reply]
It would be funny tho

**Comment by TheScreechingBagel** [comment]
but it would be funny
